yarpcerrors/errors.go

```diff
@@ -74,20 +74,14 @@ type Status struct {
 //
 // This should be used for user-defined errors.
 //
-// The name must only contain lowercase letters from a-z and dashes (-), and
-// cannot start or end in a dash. If the name is something else, an error with
-// code CodeInternal will be returned.
-//
 // Deprecated: Use only error codes to represent the type of the error.
 func (s *Status) WithName(name string) *Status {
 	// TODO: We plan to add a WithDetails method to add semantic metadata to
 	// Statuses soon.
 	if s == nil {
 		return nil
 	}
-	if err := validateName(name); err != nil {
```

Comment:
Can we delete validateName?

Reply:
It was deleted: it lived in its own file, and the file (name.go) was deleted.